packages/ERTP/src/displayInfo.js

```diff
@@ -67,6 +70,19 @@ export const coerceDisplayInfo = (allegedDisplayInfo, assetKind) => {
   assertSubset(displayInfoKeys, Object.keys(displayInfo));
   if (displayInfo.decimalPlaces !== undefined) {
     assert.typeof(displayInfo.decimalPlaces, 'number');
+    assert(
+      Number.isSafeInteger(displayInfo.decimalPlaces),
+      X`decimalPlaces ${displayInfo.decimalPlaces} is not a safe integer`,
+    );
+    assert(
+      displayInfo.decimalPlaces <= MAX_ABSOLUTE_DECIMAL_PLACES,
+      X`decimalPlaces ${displayInfo.decimalPlaces} exceeds ${MAX_ABSOLUTE_DECIMAL_PLACES}`,
+    );
+    assert(
+      displayInfo.decimalPlaces >= -MAX_ABSOLUTE_DECIMAL_PLACES,
```

Comment:
Huh. It never occurred to me that negative `decimalPlaces` are meaningful. Good to know.
(No change suggested)